source/developers-guide/attribute-system/index.md

```diff
@@ -903,3 +903,99 @@ s_articles_attributes_my_column_label = "Deutsches label"
 s_articles_attributes_my_column_supportText = "Deutscher support text"
 s_articles_attributes_my_column_helpText = "Deutscher help text"
 ```
+
+## Product slider based on attributes
+__As of Shopware 5.3__ it is easily possible to create product slider with the help of attributes and the listing widgets:
+* `Shopware/Controllers/Widgets/Listing.php` - `streamSliderAction`
+* `Shopware/Controllers/Widgets/Listing.php` - `productSliderAction`
+
+These can be used to create individual product slider. For example individual product slider for customers which will be shown in their account section. Here are two screenshots which illustrate that scenario:
+
+<img src="img/customer.jpg" alt="customer details" class="image-border" width="800px" />
+
+We choose a product stream and some variants for the customer with attribute fields.
```

Comment:
Choose a product stream and some variants for the customer with help of the attribute fields.